config/locales/hyrax.es.yml

```diff
@@ -84,22 +84,22 @@ es:
             title: Lanzamiento
             varies:
               any: El depositante puede elegir cualquier duración de embargo; se permiten arrendamientos
-              between: "El depositante puede elegir el embargo actualizado:"
-              description: "Permita que el depositante elija la configuración:"
+              between: 'El depositante puede elegir el embargo actualizado:'
+              description: 'Permita que el depositante elija la configuración:'
               embargo:
                 1yr: 1 año después del depósito
                 2yrs: 2 años después del depósito
                 3yrs: 3 años después del depósito
                 6mos: 6 meses después del depósito
                 select: Seleccione período de embargo.
-              period: "El depositante puede elegir el período de embargo hasta:"
+              period: 'El depositante puede elegir el período de embargo hasta:'
           visibility:
             description: Establecer políticas de visibilidad para el conjunto administrativo. Establecer políticas de embargo de honor arriba.
-            everyone: "Público: el depositante solo puede elegir la configuración de visibilidad pública"
-            institution: "Institución: el depositante solo puede seleccionar la configuración de visibilidad de la institución"
-            restricted: 'Privado: el depositante solo puede seleccionar privado para visibilidad. El acceso está restringido a los administradores de repositorios, gerentes y espectadores del conjunto. Se debe usar con el ajuste "Sin embargo" anterior.'
+            everyone: 'Público: el depositante solo puede elegir la configuración de visibilidad pública'
+            institution: 'Institución: el depositante solo puede seleccionar la configuración de visibilidad de la institución'
+            restricted: 'Privado: el depositante solo puede seleccionar privado para visibilidad. El acceso está restringido a los administradores de repositorios, gerentes y espectadores del conjunto. Se debe usar con el ajuste 'Sin embargo' anterior.'
```

Comment:
items with single quotes need to be wrapped in double quotes

Reply:
Is this saying to add double quotes only to the 'Sin embargo' on line 100 or add double quotes to the lines 98-100? Sorry for so many questions, I would like to get this correct and it's not entirely clear to me what this feedback is saying.